omnistrate.operator.yaml

```
@@ -0,0 +1,1635 @@
+            app.kubernetes.io/managed-by: Helm
+            app.kubernetes.io/instance: cluster
+            app.kubernetes.io/version: 0.17.0
+            app.kubernetes.io/component: middleware
+          annotations:
+            
+        spec:
+          clusterSize: 3
+          persistenceEnabled: true
+          clusterVersion: v7
+          redisLeader:
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)